Scripts/buildnwnx.sh

```diff
@@ -43,7 +43,9 @@ pushd ./build-nwnx
 
 cmake -D CMAKE_BUILD_TYPE=$BUILD_TYPE $SANITIZE ..
 
-make ${JOBS} all
+if ! make ${JOBS} all; then 
+  exit 1
+fi
```

Comment:
Doesn't this clobber the value and return 1 always? Not what the commit message says?
Also, not related to this PR, would be best as separate.

Reply:
Yes, it does clobber. I can do it in an other MR